src/features/subdomain-table/SubdomainTableRow.tsx

```diff
@@ -0,0 +1,67 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Library Imports
+import { ethers } from 'ethers';
+import { TokenPriceInfo } from '@zero-tech/zns-sdk';
+
+//- Hook Imports
+import { useDomainMetadata } from '../../lib/hooks/useDomainMetadata';
+import { useDomainMetrics } from '../../lib/hooks/useDomainMetrics';
+import { formatEthers, formatNumber } from '../../lib/util/number/number';
+import { useBuyNowPrice } from '../../lib/hooks/useBuyNowPrice';
+
+type SubdomainTableRowProps = {
+	domainId: string;
+	domainName: string;
+	domainMetadataUri: string;
+	paymentTokenData: TokenPriceInfo;
+	onClick: (domainName: string) => void;
+};
+
+const SubdomainTableRow: FC<SubdomainTableRowProps> = ({
+	domainId,
+	domainName,
+	domainMetadataUri,
+	paymentTokenData,
+	onClick,
+}) => {
+	const { data: domainMetrics } = useDomainMetrics(domainId);
```

Comment:
metrics is currently causing a 429 'Too many requests error'. Would you have suggestion on how to handle this? I was unable to get to the bottom of the cause

Reply:
<img width="1485" alt="Screenshot 2022-08-11 at 16 29 54" src="https://user-images.githubusercontent.com/39112648/184171394-c28770be-6090-457d-a3b2-56e0c006eb5a.png">

Reply:
@domw30 it will be because there's no pagination on that table, so it's loading metrics for all subdomains at once. Just for testing, render `subdomains.splice(0, 10)` so we can get an idea that it's working without hitting the dreaded 429.

Reply:
Okay cool. Tested and works without hitting 429 - up to around 60 results-ish